main.js

```diff
@@ -0,0 +1,86 @@
+// let name = document.location.pathname.slice(1);
+if (!name) {
+  console.log("no place name specfied");
+} else {
+  console.log("fetching place named '" + name + "'");
+
+  fetch(name + ".json")
+    .then(function(response) {
+      return response.text();
+    })
+    .then(function(myText) {
+      console.log(myText);
+    });
+}
+
+function getRestaurantNames() {
+  fetch("all.json")
+    .then(response => response.json())
+    .then(object => {
+      console.log(object)
+    });
+};
+
+getRestaurantNames();
+
+function getWebsite(address) {
+  // fetch(name + ".json");
+}
+
+let getLatLong = function(address) {
```

Comment:
getLatLong is a bit of a misnomer since the end product of this function isn't a lat long, but is a marker added to the map. Calling it something like `addMarker` would be more accurate.